samples/basic/servo_motor/prj_bbc_microbit.conf

```diff
@@ -2,4 +2,3 @@ CONFIG_STDOUT_CONSOLE=y
 CONFIG_PRINTK=y
 CONFIG_PWM=y
 CONFIG_PWM_NRF5_SW=y
-CONFIG_PWM_NRF5_SW_0_CLOCK_PRESCALER=3
```

Comment:
After this change, the whole file seems to be unneeded. `CONFIG_PWM_NRF5_SW` is `y` by default, when PWM is enabled.

Reply:
You are correct. Will remove file